### PR TITLE
Fix for Qt 15.5 Compilation

### PR DESCRIPTION
**Pull request by breadsax**
Code changes by ririen and bedri to fix Qt 5.15 compilation. Tested that it works on Mac OS Mojave.

### CODE REVIEW

**Comment by FryDay** [comment]
Is there an ETA on getting this merged? I am the maintainer of the Arch Linux AUR package and had to add 2 patches to the package in order to compile Ravencoin on linux.

File: src/qt/trafficgraphwidget.cpp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #include "clientmodel.h"
 
 #include <QPainter>
+#include <QPainterPath>
```
Comment (FryDay) [inline src/qt/trafficgraphwidget.cpp:10]: You don't need this here since you include it in the header.

File: src/qt/trafficgraphwidget.h [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 #include <QWidget>
 #include <QQueue>
+#include <QPainterPath>
```
Comment (FryDay) [inline src/qt/trafficgraphwidget.h:11]: You should only need this in either the header or the cpp. Not both.

**Comment by spyder46n2** [comment]
Looks like we have this code in now. Thanks!

**Pull request closed without merging**